src/Essentials/src/Geolocation/Geolocation.android.cs

```diff
@@ -112,6 +117,84 @@ void RemoveUpdates()
 			}
 		}
 
+		public async Task<bool> StartListeningForegroundAsync(GeolocationListeningRequest request)
+		{
+			_ = request ?? throw new ArgumentNullException(nameof(request));
```

Comment:
```suggestion
			ArgumentNullException.ThrowIfNull(nameof(request));
```

Reply:
I checked the Maui project, and `ArgumentNullException.ThrowIfNull()` is not used much, but I will do the change in Geolocation. I like it, it's more elegant.

Reply:
It seems the syntax actually is `ArgumentNullException.ThrowIfNull(request);` and the nameof(request) in the second parameter can be omitted and is derived from the first parameter.